libstuff/STCPServer.cpp

```diff
@@ -23,15 +23,25 @@ STCPServer::Port* STCPServer::openPort(const string& host) {
     return &*portIt;
 }
 
-void STCPServer::closePorts() {
+void STCPServer::closePorts(list<Port*> except) {
+    for (auto p : except) {
```

Comment:
Can you either add a comment here and separate it from the next block by a blank line, or merge this into the next block by putting it under its comment?  (See [here](https://github.com/Expensify/Style-Guide/blob/master/php.md#blank-lines))